en/middleware/micrortps.md

```diff
@@ -1,44 +1,47 @@
+
+*Fast DDS* is a very lightweight cross-platform implementation of the latest version of the RTPS protocol / DDS middleware.
+It was previously named *Fast RTPS*.
 :::
 
-RTPS has been adopted as the middleware for the ROS2 (Robot Operating System).
-The *Fast RTPS bridge* allows us to better integrate with ROS2, making it easy to share sensor values, commands, and other vehicle information.
+:::note
+RTPS has been adopted as the middleware for the ROS 2 (Robot Operating System).
 
-This topic describes the RTPS bridge architecture (and how it is used within the ROS2/ROS application pipeline).
-It also shows how to compile needed code to:
-1. Write a simple *Fast RTPS* application to subscribe to PX4 changes
-1. Connect ROS2 nodes with PX4 (via the RTPS Bridge, and using the `px4_ros_com` package)
-1. Connect ROS (ROS "version 1") nodes with PX4 by additionally using the `ros1_bridge` package to bridge ROS2 and ROS.
+For information about how to use this interface within the ROS 2 applications and development workflows, see [PX4-ROS 2 bridge](../ros/ros2_comm.md).
+:::
 
 
 ## When should RTPS be used?
 
-RTPS should be used when you need to reliably share time-critical/real-time information between the flight controller and off board components.
-In particular it is useful in cases where off-board software needs to become a *peer* of software components running in PX4 (by sending and receiving uORB topics).
+RTPS should be used when you need to reliably share time-critical/real-time information between the flight controller and offboard components.
+In particular it is useful in cases where offboard software needs to become a *peer* of software components running in PX4 (by sending and receiving uORB topics).
 
-Possible use cases include communicating with robotics libraries for computer vision, and other use cases where real time data to/from actuators and sensors is essential for vehicle control.
+Possible use cases include communicating with robotics libraries for computer vision, and other use cases where real-time data to/from actuators and sensors is essential for vehicle control.
 
 :::note
-*Fast RTPS* is not intended as a replacement for MAVLink.
-MAVLink remains the most appropriate protocol for communicating with ground stations, gimbals, cameras, and other offboard components (although *Fast RTPS* may open other opportunities for working with some peripherals).
+*Fast DDS* is not intended as a replacement for MAVLink.
+MAVLink remains the most appropriate protocol for communicating with ground stations, gimbals, cameras, and other offboard components (although *Fast DDS* may open other opportunities for working with some peripherals).
 :::
 
 :::tip
 RTPS can be used over slower links (e.g. radio telemetry), but care should be taken not to overload the channel.
 :::
 
+
 ## Architectural overview
 
-### RTPS Bridge
+### microRTPS Bridge
```

Comment:
Naming inconsistency here and elsewhere - aren't we now calling this "PX4-Fast RTPS(DDS) Bridge"? 

Anyway, I think this should be deleted - haven't done that so I could make this comment!

Reply:
@TSC21 Can you close the remaining issues, like this one?

Reply:
Let's use microRTPS yes.